setup.cfg

```diff
@@ -32,17 +32,18 @@ install_requires =
     paramiko==3.4.0
     ruamel.yaml
     furl
-    pydantic[email]>=1.8.2,<2.0.0
```

Comment:
This should have a > 2.0.0 now as otherwise it will break for people with older deployments

Reply:
ok, i will fix pydantic[email] now...